x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/empty/index.tsx

```diff
@@ -21,7 +21,8 @@ export const EventFiltersListEmptyState = memo<{
   onAdd: () => void;
   /** Should the Add button be disabled */
   isAddDisabled?: boolean;
-}>(({ onAdd, isAddDisabled = false }) => {
+  backComponent?: React.ReactNode;
+}>(({ onAdd, isAddDisabled = false, backComponent }) => {
   return (
     <ManagementEmptyStateWraper>
```

Comment:
**nit:** This is not in this PR but needs an extra `p` in `...Wraper`. If you do happen to update this then would be nice to have the file names updated as well.